extensions/wikia/CommunityPage/CommunityPageSpecialHooks.class.php

```diff
@@ -107,6 +111,10 @@ public static function onUserFirstEditOnLocalWiki( $userId, $wikiId ) {
 		WikiaDataAccess::cachePurge( $key );
 		CommunityPageSpecialUsersModel::logUserModelPerformanceData( 'purge', 'recently_joined' );
 
+		// Set cookie to show first edit modal to user
+		global $wgCookieDomain;
+		setcookie ( self::FIRST_EDIT_COOKIE_KEY, true, time()+60, '/', $wgCookieDomain );
```

Comment:
Remove the space between the function name and the opening parenthesis.

Also, the path should be set to `$wgCookiePath` rather than hardcoded as `/`.